package.json

```diff
@@ -13,19 +13,24 @@
     "angular-ui-notification": "0.1.0",
     "angular-ui-router": "1.0.0-alpha.1",
     "lodash": "2.4.2",
-    "ngreact": "0.2.0",
+    "ngreact": "^0.4.1",
```

Comment:
Please stick to exact versions. I recommend to set it once with `npm config set save-exact true`.